### PR TITLE
[DATA-2708] fix: prevent-overwrite

### PR DESCRIPTION
**Pull request by MHAbido**

### CODE REVIEW

File: pre_commit_config_updater.py [hunk 1]
```diff
@@ -1,6 +1,22 @@
 import argparse
+import shutil
 import urllib.request
 
+import toml
+
+
+def init_toml(toml_file: str):
```
Comment (RavijitRamana) [inline pre_commit_config_updater.py:8]: can there be a case that we use this pre-commit config updater where this file doesn't exist?

Comment (MHAbido) [reply pre_commit_config_updater.py:8]: Nope

Comment (RavijitRamana) [reply pre_commit_config_updater.py:8]: I think it is a pep standard now but are we sure it always exists.

Comment (MHAbido) [reply pre_commit_config_updater.py:8]: yes it's standard now and even if it's not there it will be auto created for `issort` and `black` config.

**Review by RavijitRamana — APPROVED** [review]

**Pull request merged**